servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/MutualSslTest.java

```diff
@@ -44,32 +54,58 @@
     public final Timeout timeout = new ServiceTalkTestTimeout();
     private final SslProvider serverSslProvider;
     private final SslProvider clientSslProvider;
+    @SuppressWarnings("rawtypes")
+    private final Map<SocketOption, Object> serverListenOptions;
+    @SuppressWarnings("rawtypes")
+    private final Map<SocketOption, Object> clientOptions;
 
-    public MutualSslTest(final SslProvider serverSslProvider, final SslProvider clientSslProvider) {
+    public MutualSslTest(final SslProvider serverSslProvider, final SslProvider clientSslProvider,
+                         @SuppressWarnings("rawtypes") final Map<SocketOption, Object> serverListenOptions,
+                         @SuppressWarnings("rawtypes") final Map<SocketOption, Object> clientOptions) {
         this.serverSslProvider = serverSslProvider;
         this.clientSslProvider = clientSslProvider;
+        this.serverListenOptions = serverListenOptions;
+        this.clientOptions = clientOptions;
     }
 
-    @Parameterized.Parameters(name = "server={0} client={1}")
+    @Parameterized.Parameters(name = "server={0} client={1} server opts={2} client opts={3}")
     public static Collection<Object[]> sslProviders() {
         return asList(
-                new Object[]{JDK, JDK},
-                new Object[]{JDK, OPENSSL},
-                new Object[]{OPENSSL, JDK},
-                new Object[]{OPENSSL, OPENSSL}
+                new Object[]{JDK, JDK, emptyMap(), emptyMap()},
```

Comment:
Does this assume that fast open will always be available in the test environment?

Reply:
no it won't be but the motivation for testing all the combinations is that even if fast open isn't supported it asking for it shouldn't break anything (provides an opportunity for optimization).

Reply:
OK, as long as it isn't required for successful testing.